From d0a294363356272fa07be52602e4ba0a7ffd5790 Mon Sep 17 00:00:00 2001
From: 5656565566 <contact@home56.top>
Date: Sat, 29 Aug 2026 14:24:18 +0800
Subject: [PATCH] build/soong: make cc installable arch-variant

Allow installability to be selected per architecture variant. NativeBridge libc must remain available for dependency resolution while its host-side output is excluded from installation.

Change-Id: I861f384cb32fa80d3b542ed1fb33db57db6ba368
---
 cc/cc.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cc/cc.go b/cc/cc.go
index 68c9e92e0..afc759480 100644
--- a/cc/cc.go
+++ b/cc/cc.go
@@ -336,7 +336,7 @@ type BaseProperties struct {
 	SnapshotSharedLibs  []string `blueprint:"mutated"`
 	SnapshotRuntimeLibs []string `blueprint:"mutated"`
 
-	Installable *bool
+	Installable *bool `android:"arch_variant"`
 
 	// Set by factories of module types that can only be referenced from variants compiled against
 	// the SDK.
-- 
2.47.3

